packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -70,26 +75,61 @@ export const Combobox = ({ options, onChange, value, ...restProps }: ComboboxPro
       }
     },
   });
+
+  // the order of middleware is important!
+  const middleware = [
+    flip({
+      // see https://floating-ui.com/docs/flip#combining-with-shift
+      crossAxis: false,
+      boundary: document.body,
+      fallbackPlacements: ['top'],
```

Comment:
Are there cases where we want it to fallback to the sides?

Reply:
I cannot think of any use case that needs this, but there might be.

Reply:
fallback to top is correct. We don't want a drop down appearing to the right of the input